src/scss/utilities/_utilities.scss

```diff
@@ -1,5 +1,11 @@
 @import "~nhsuk-frontend/packages/core/settings/colours";
 // re-usable styles and mixins
+
+h2 {
```

Comment:
The `h2` tag is used somewhere else (eg. notes-drawer, confirm-recommendation, etc.), and the `notes-drawer.component.scss` also has the same h2 style. Should we remove that one?